static/app/views/issueDetails/streamline/sidebar/seerDrawer.tsx

```diff
@@ -85,13 +88,74 @@ export function SeerDrawer({group, project, event}: SeerDrawerProps) {
     }
   };
 
+  const scrollToSection = useCallback(
+    (sectionType: string | null) => {
+      if (!scrollContainerRef.current || !autofixData) {
+        return;
+      }
+
+      const findStepByType = (type: string) => {
+        if (!autofixData?.steps?.length) {
+          return null;
+        }
+        const step = autofixData.steps.find(s => {
+          if (type === 'root_cause')
+            return s.type === AutofixStepType.ROOT_CAUSE_ANALYSIS;
+          if (type === 'solution') return s.type === AutofixStepType.SOLUTION;
+          if (type === 'code_changes') return s.type === AutofixStepType.CHANGES;
+          return false;
+        });
+        return step;
+      };
+
+      if (sectionType) {
+        const step = findStepByType(sectionType);
+        if (step) {
+          const elementId = `autofix-step-${step.id}`;
+          const element = document.getElementById(elementId);
+          if (element) {
+            element.scrollIntoView({behavior: 'smooth'});
+            userScrolledRef.current = true;
+
+            // Clear the scrollTo parameter from the URL after scrolling
+            // This allows automatic scrolling to continue working for future updates
+            setTimeout(() => {
+              navigate(
+                {
+                  pathname: location.pathname,
+                  query: {
+                    ...location.query,
+                    scrollTo: undefined,
+                  },
+                },
+                {replace: true}
+              );
+            }, 200);
+          }
+        }
+      }
+    },
+    [autofixData, location, navigate]
+  );
+
   useEffect(() => {
     // Only auto-scroll if user hasn't manually scrolled
     if (!userScrolledRef.current && scrollContainerRef.current) {
       scrollContainerRef.current.scrollTop = scrollContainerRef.current.scrollHeight;
     }
   }, [autofixData]);
 
+  useEffect(() => {
+    const scrollTo = location.query.scrollTo as string | undefined;
+    if (scrollTo) {
+      const timeoutId = setTimeout(() => {
+        scrollToSection(scrollTo);
+      }, 100);
+      return () => clearTimeout(timeoutId);
+    }
+    return () => {};
+  }, [location.query.scrollTo, scrollToSection, autofixData]);
```

Comment:
if `autofixData` changes, will we just keep calling this effect?

Reply:
good catch, removed that dependency